src/util/bip158.rs

```diff
@@ -103,14 +103,8 @@ impl From<io::Error> for Error {
 
 /// a computed or read block filter
 pub struct BlockFilter {
-    /// id of the block
-    pub block_hash: sha256d::Hash,
-    /// filte type (see SCRIPT_FILTER)
-    pub filter_type: u8,
```

Comment:
Can we remove the `SCRIPT_FILTER` constant now too?

Reply:
yes, removed